cmake/onnxruntime_opschema_lib.cmake

```diff
@@ -0,0 +1,28 @@
+# Schema library (for contrib ops and training ops)
+
+set (CONTRIB_OPS_DIR ${ONNXRUNTIME_ROOT}/core/graph/contrib_ops)
+set (TRAINING_OPS_DIR ${ORTTRAINING_ROOT}/orttraining/core/graph)
+
+file(GLOB_RECURSE contrib_ops_schema_src
+   "${CONTRIB_OPS_DIR}/*.cc"
+   "${TRAINING_OPS_DIR}/training_op_defs.cc"
+)
+
+add_library(ort_opschema_lib ${contrib_ops_schema_src})
+
+# ${CMAKE_CURRENT_BINARY_DIR} is so that #include "onnxruntime_config.h" is found
+target_include_directories(ort_opschema_lib PRIVATE ${ONNXRUNTIME_ROOT} ${ORTTRAINING_ROOT} ${CMAKE_CURRENT_BINARY_DIR})
+onnxruntime_add_include_to_target(ort_opschema_lib onnxruntime_common onnx onnx_proto protobuf::libprotobuf flatbuffers)
```

Comment:
Is it possible to make it not depend on onnxruntime_common?

Reply:
Unfortunately, there were some dependencies. One is GetStackTrace(), which is used in common macros. Any suggestions?

Reply:
Sorry I don't have any now. We may reconsider it later.